shared/constants/multichain/assets.ts

```diff
@@ -13,3 +14,17 @@ export enum MultichainNativeAssets {
   SOLANA_DEVNET = `${MultichainNetworks.SOLANA_DEVNET}/slip44:501`,
   SOLANA_TESTNET = `${MultichainNetworks.SOLANA_TESTNET}/slip44:501`,
 }
+
+/**
+ * Maps network identifiers to their corresponding native asset types.
+ * Each network is mapped to an array containing its native asset for consistency.
+ */
+export const NETWORK_ASSET_MAP: Record<MultichainNetworks, CaipAssetType[]> = {
```

Comment:
Could you rename this to `MULTICHAIN_NETWORK_TO_ASSET_TYPES`?

```suggestion
export const MULTICHAIN_NETWORK_TO_ASSET_TYPES: Record<MultichainNetworks, CaipAssetType[]> = {
```